src/website/src/app/documentation/demos/tabs/tabs-angular-vertical.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+
+const EXAMPLE = `
+<clr-tabs clrTabsLayout="vertical">
```

Comment:
@youdz do we want this, or `clrTabLayout` or `clrLayout`? We use `clrLayout` with forms.

Reply:
I don't think we have a conflict risk here, so I like `clrLayout` better. Like you I guess, since you raised it.

Reply:
Done

Reply:
Oops, I'll have to unresolve this. The clrLayout we use in the tabs is a `@Directive`, not an `@Input` as it it here. So it tries to get applied on the tabs too, leading to many errors.
I guess we can't use clrLayout then.

Reply:
Done again after supplying a more specific selector to the forms clrLayout directive.